src/components/Uptime.vue

```diff
@@ -75,6 +75,14 @@ export default {
 
             return "";
         },
+
+        title() {
+            if (this.type === "720") {
```

Comment:
What is `720`??

Reply:
720 is the 30 days type (24x30).
Both types (24 and 720) are used at https://github.com/louislam/uptime-kuma/blob/839183aa8542fe484ed5e697ebb479502469c33a/src/pages/Details.vue#L73